Scuti/Scripts/UI/MyAccountPresenter.cs

```diff
@@ -28,6 +28,14 @@ protected void OnSetData()
             locationText.text = Data.Location;
         }
 
+        public override void Open()
+        {
+            Debug.Log("Open Account presenter");
```

Comment:
@Jpablogs8 please remove this log and any other logs. Please look for logs when you create PRs